### DISCUSSION TITLE
开发计划（2024 年 6 月）

### DISCUSSION THREAD

**Discussion by cloudwu** [ejoy/ant#165] [Announcements]
由于一些变故（ blog [重新启程](https://blog.codingnow.com/2024/05/farewell.html) ），Ant 引擎开发暂停了一小段时间。[Vaststars](https://github.com/ejoy/vaststars) 目前暂停开发，但我想设计一个基于 Steam 平台的新游戏，并使用 Ant Engine 开发。

所以，Ant 的后续开发计划主要是跟随新的游戏 Demo 的需求而来。

* [ ]  [新的点选系统](https://github.com/ejoy/ant/discussions/155)。
* [ ] 加强预制形状的支持，例如平板、立方体等，方便制作游戏 demo 。会和 [重构资源管理中的 mesh 管理](https://blog.codingnow.com/2024/06/asset_mgr_rework.html) 一起做。
* [ ] 增加一个 2D 空间到 3D 空间的映射系统。方便用 3d 模型搭建 2d 逻辑的场景。

**Comment by labspc** [comment]
> 我想设计一个基于 Steam 平台的新游戏，并使用 Ant Engine 开发 

支持云大的这个想法，要是能做一个很小的demo游戏作为 ant wiki的补充就很nice啊。

**Comment by constly** [comment]
我希望未来test目录下能有一些比较漂亮/正式的场景展示
（由于我之前没研究过渲染，对于怎样才能让我的demo更漂亮感觉难以下手，所以我最近准备先学习下bgfx以及shader = =）

**Reply by actboy168** [reply]
可以考虑复刻bgfx的example，至少美术资源可以复用了。

**Reply by constly** [reply]
有道理！
